scout/server/blueprints/variants/controllers.py

```diff
@@ -83,6 +84,18 @@ def populate_force_show_unaffected_vars(institute_obj, form):
         form.show_unaffected.data = True
 
 
+def set_persistent_filters_choices(
```

Comment:
👍  Nice with a function, but naming is tricky. I might have stuck with populate, since it is ambiguous, but I guess more clearly